sig-network/README.md

```diff
@@ -135,6 +148,31 @@ The following [subprojects][subproject-definition] are owned by sig-network:
 [subproject-definition]: https://github.com/kubernetes/community/blob/master/governance.md#subprojects
 [working-group-definition]: https://github.com/kubernetes/community/blob/master/governance.md#working-groups
 <!-- BEGIN CUSTOM CONTENT -->
+
+### Subproject Leads
+
+SIG network provides some additional responsibilities for subproject leads beyond was in covered in the [standard subproject definition][subprojects].
+Most of these additional responsibilities relate to communication about their projects within SIG Network, and to the greater community:
+
+- **Transparent Project Planning, Maintenance and Communication:**
+  - Subproject Leads **MUST** provide transparent view into their historical and future project plans (eg; using GitHub project boards, KEPs, or custom enhancement proposals (see [GEPs]/[NPEPs])).
```

Comment:
Sorry, habit. Inspired by RFCs and a common [thing we've done in Gateway API](https://github.com/kubernetes-sigs/gateway-api/blob/ce6d2c6366847386747d7aa6ebb8ff95693dd536/apis/v1/gateway_types.go#L70), similarly inspired. It's a good callout that if someone comes in here and doesn't know that, it might seem a bit nasty. I'm fine with just changing all these to a plain must.

Reply:
In RFCs it's always applied to _implementations_ though. It feels weird to apply it to people.

Reply:
Agreed

Reply:
ack changing it to must